src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -192,7 +193,7 @@ public final Connection getConnection(final long hardTimeout) throws SQLExceptio
             final long now = clockSource.currentTime();
             if (bagEntry.evicted || (clockSource.toMillis(now - bagEntry.lastAccess) > ALIVE_BYPASS_WINDOW_MS && !isConnectionAlive(bagEntry.connection))) {
                closeConnection(bagEntry, "connection evicted or dead"); // Throw away the dead connection and try again
-               timeout = hardTimeout - clockSource.elapsedTimeMs(startTime);
```

Comment:
I find the *elapsedTimeMs()* form more readable, and performance-wise this method gets inlined anyway so the two are equivalent.